quiz/backend/users/migrations/0003_alter_customuser_username.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.1.7 on 2023-03-31 11:30
+
+import django.contrib.auth.validators
+from django.db import migrations, models
+import helpers.validators
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0002_superuser"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="customuser",
+            name="username",
+            field=models.CharField(
+                max_length=100,
+                validators=[
```

Comment:
after migration, another one is being created

0004_alter_customuser_username.py:
```
class Migration(migrations.Migration):
    dependencies = [
        ("users", "0003_alter_customuser_username"),
    ]

    operations = [
        migrations.AlterField(
            model_name="customuser",
            name="username",
            field=models.CharField(
                blank=True,
                max_length=100,
                validators=[
                    helpers.validators.validate_profanity,
                    django.contrib.auth.validators.UnicodeUsernameValidator(),
                ],
            ),
        ),
    ]
```

Reply:
What are your suggestions? Should I squash it with custom migration I'd rather not😅

Reply:
Ok now I see the problem, god I have really negative thoughts about rebasing.😅

Reply:
your migration is missing one field (how?)

```
                blank=True,

```
 so I think if you delete it and create it again it should be fine 😛

Reply:
So migrations should work now pull request is now rebased on top of master